tutorials/04-switching-physics-engines.md

```diff
@@ -0,0 +1,99 @@
+\page switchphysicsengines Physics engines
+
+This tutorial describes how to switch between physics engines in Ignition Physics when using Ignition Gazebo.
```

Comment:
Consider keeping the lines to 80 characters so git can generate better diffs.

Reply:
@anindex  friendly ping. This comment doesn't seem to be addressed yet.

Reply:
Yeah I forgot sorry :)